### PR TITLE
fix for undefined selected option

### PR DESCRIPTION
**Pull request by marijahlobik**
I added check if none option is selected (or there is no property selected = true)

### CODE REVIEW

File: src/components/LeftNavigation/LeftNavigation.tsx [hunk 1]
```diff
@@ -78,7 +78,10 @@ export class LeftNavigation extends CommonComponent<ILeftNavigationProps, any> {
 
     private getSelectedId(options: Array<ILeftNavigationOption>) {
         const selectedOptions = options.filter(option => { return option.selected ? option.id : null; });
-        return selectedOptions[0].id;
+        
+        if (selectedOptions.length > 0) {
+            return selectedOptions[0].id;
+        }
```
Comment (hbagarich) [inline src/components/LeftNavigation/LeftNavigation.tsx]: Can you explicitly return some value from function?

File: src/components/LeftNavigation/LeftNavigation.tsx [hunk 2]
```diff
@@ -78,7 +78,10 @@ export class LeftNavigation extends CommonComponent<ILeftNavigationProps, any> {
 
     private getSelectedId(options: Array<ILeftNavigationOption>) {
```
Comment (hbagarich) [inline src/components/LeftNavigation/LeftNavigation.tsx]: Can you declare which type should this function return?

Comment (marijahlobik) [reply src/components/LeftNavigation/LeftNavigation.tsx]: I did it now

File: src/components/LeftNavigation/LeftNavigation.tsx (hunk 1 above)
Comment (marijahlobik) [reply src/components/LeftNavigation/LeftNavigation.tsx]: I returned empty string (because id is defined as string)

**Pull request merged**